Shared/Samples/Clip geometry/ClipGeometryView.swift

```diff
@@ -91,6 +39,70 @@ struct ClipGeometryView: View {
     }
 }
 
+private extension ClipGeometryView {
+    private class Model: ObservableObject {
+        /// A map with a topographic basemap style and an initial viewpoint of Colorado.
+        let map: Map = {
+            let map = Map(basemapStyle: .arcGISTopographic)
+            // Sets the initial viewpoint to Colorado and adds additional padding.
+            map.initialViewpoint = Viewpoint(targetExtent: .coloradoEnvelope.expanded(by: 2))
+            return map
+        }()
+        
+        /// The graphics overlay containing an unclipped graphic of Colorado.
+        private let coloradoGraphicsOverlay = GraphicsOverlay(
+            graphics: [
+                Graphic(geometry: .coloradoEnvelope, symbol: .coloradoFill)
+            ]
+        )
+        
+        /// The graphics overlay containing graphics of the other envelopes, outlined by a dotted red line.
+        private let envelopesGraphicsOverlay = GraphicsOverlay(
+            graphics: Geometry.envelopes.map {
+                Graphic(geometry: $0, symbol: SimpleLineSymbol(style: .dot, color: .red, width: 3))
+            }
+        )
+        
+        /// The graphics overlay containing clipped graphics.
+        private let clippedGraphicsOverlay = GraphicsOverlay()
+        
+        /// The unclipped graphic of Colorado.
+        private var coloradoGraphic: Graphic { coloradoGraphicsOverlay.graphics.first! }
+        
+        /// The graphics overlays used in this sample.
+        var graphicsOverlays: [GraphicsOverlay] {
```

Comment:
In the following samples, if one uses multiple graphics overlays, I use a computed property to expose them to the view, instead of enumerate them such as model.overlay1, model.overlay2, etc.